package.json

```diff
@@ -13,10 +13,12 @@
     "@vitejs/plugin-react": "^1.3.2",
     "@vitejs/plugin-react-refresh": "^1.3.6",
     "antd": "^5.8.2",
+    "axios": "^1.7.8",
```

Comment:
why not use `fetch` directly?

ref: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API